docs/core/deploying/single-file.md

```diff
@@ -11,6 +11,10 @@ Bundling all application-dependent files into a single binary provides an applic
 
 Single File deployment is available for both the [framework-dependent deployment model](index.md#publish-framework-dependent) and [self-contained applications](index.md#publish-self-contained). The size of the single file in a self-contained application will be large since it will include the runtime and the framework libraries. The single file deployment option can be combined with [ReadyToRun](ready-to-run.md) and [Trim (an experimental feature in .NET 5.0)](trim-self-contained.md) publish options.
 
+## Output differences from .NET 3.x
+
+In .NET Core 3.x, publishing as a single file produced exactly one file, with all files next to the app bundled and extracted at startup. In .NET 5+, there is no extraction by default. On Windows, this can result in multiple files in the output: the app file, and the core runtime DLLs alongside. To embed those files for extraction, like in .NET Core 3.x, set the property `IncludeNativeLibrariesForSelfExtract` to `true`. For more information about extraction, see [Other considerations](#other-considerations).
```

Comment:
I think this needs to be clarified. What is the experience exactly? It sounds like `PublishSingleFile` setting does nothing in 5.0, but I don't think that is correct. The bit about `IncludeNativeLibrariesForSelfExtract` doesn't seem to make sense either. Below in **Other considerations**, this setting is talking about linux only, yet up here this is speaking in generalities.

Reply:
Added more explanation -- does this make sense? Or am I still missing something?

Reply:
I think part of my confusion is `In .NET 5+, there is no extraction by default.` this sounds like you're getting a single file out but it wont self extract.